### ISSUE TITLE
dotnet-ef stops working after tinkering with later versions

### ISSUE THREAD

**Issue by shaulbehr** [dotnet/efcore#15860]
I have a project that is using EF Core 2.1. I created a code branch that attempted to upgrade to EF Core 2.2, found that required also upgrading to NET Core 2.2, and that went down a rabbit hole. Put that branch on ice and switched back to my old code.

Now I'm trying to run EF migrations from the command line, and it ain't happy:

> dotnet ef migrations add "My Migration" 

> The application to execute does not exist: 'C:\Users\Shaul\.dotnet\tools\.store\dotnet-ef\2.1.11-servicing-32099\dotnet-ef\2.1.11-servicing-32099\tools\netcoreapp2.1\any\dotnet-ef.dll'.

I uninstalled and reinstalled ef-migrations with the command

> dotnet tool install --global --version 2.1.11 dotnet-ef

But for some reason the command is looking for the application in a `2.1.11-servicing-32099` directory, instead of just plain `2.1.11`.

How to fix this?

### Further technical details
EF Core version: 2.1
Database Provider: Npgsql
Operating system: Windows 10
IDE: JetBrains Rider

**Comment by ajcvickers** [comment]
@shaulbehr Note that (prior to 3.0) the ef command is not really a global tool, but instead is included int the SDK. So the tool version that is being used should come from the SDK that you are using. Also, the tool is backward compatible down to 2.0, so even if you still have the 2.2 SDK installed and are using it to build a 2.1 application, the EF tool should still work.

I don't know why you are seeing what you are seeing? Are you able to reproduce in a small project that you can post? Also, what SDKs to you have installed?

**Comment by shaulbehr** [comment]
@ajcvickers This isn't something I can repro in different branches. The point being that I attempted an upgrade to 2.2 in one branch, then switched back to a 2.1 branch, and found that EF was looking for the binaries in a non-existent directory.
I worked around the problem simply by copying the binaries from my `2.1.1` directory to a new `2.1.1-servicing-32099` directory. Bit of a hack, but at least it solves my problem for now.
